designs/manager_policy_mapping/policy_mapping.md

```diff
@@ -0,0 +1,76 @@
+# Policy mapping workflow with IP/Mac
+
+This document describes the proposed changes for policy mapping workflow with IP/Mac.
+
+## Background
+
+Currently, we have two ways to determine which policy gets applied to which workload:
+
+- Provide manager with encapsulation tag
+- Provide manager with IP/Mac
+
+If the manager is supporting policy mapping and security group, we provide manager security group information to bind the policy.
+
+## Constraints and Assumptions
+The Security Group on the manager with one or more policies/interfaces is never deleted.
```

Comment:
The appliance security manager **will not delete** a security group with one or more security group interfaces.